phpstan-baseline.neon

```diff
@@ -7428,7 +7428,7 @@ parameters:
 		-
 			message: "#^Parameter \\#1 \\$themeName of class Shopware\\\\Storefront\\\\Theme\\\\Exception\\\\ThemeCompileException constructor expects string, string\\|null given\\.$#"
 			count: 1
-			path: src/Storefront/Theme/ThemeCompiler.php
+			path: src/Storefront/Theme/PhpStyleCompiler.php
```

Comment:
best would be of course fixing this error instead of changing the baseline file :grimacing:

Reply:
Sry, but did I do the null handling wrong?

Reply:
no, sorry that I bothered you...